uber/jinja.py

```diff
@@ -0,0 +1,61 @@
+from uber.common import *
+
+
+class JinjaEnv:
+    _env = None
+    _exportable_functions = {}
+    _filter_functions = {}
+    _template_dirs = []
+
+    @staticmethod
```

Comment:
Here are some general notes about the difference between ``@staticmethod`` and ``@classmethod`` and why you'd use one over the other.  I wouldn't necessarily bother changing what you have here, especially given that most of what I'm about to say doesn't apply, so this is really more of a "for future reference" kind of thing.

Consider two different class implementations:

```python
class Adder:
    to_add = 5

    @staticmethod
    def add(x):
        return x + Adder.to_add

class Subtracter:
    to_subtract = 5

    @classmethod
    def subtract(cls, x):
        return x - cls.to_subtract
```

These two classes are basically the same, except that one is using ``@staticmethod`` and referring to ``Adder.to_add`` directly, while the other is using a ``@classmethod`` and referring to ``cls``.  The latter is more Pythonic, because it allows us to subclass ``Subtracter`` and allow the different class instances to refer to their own class fields.

None of that probably matters much in this case, since I doubt that a plugin would want or need to subclass ``JinjaEnv``, so the advantages are mostly irrelevant.